src/cmd/repo.go

```diff
@@ -25,6 +26,9 @@ Example: your SSH identity should be something like
 		script.OpenBrowser(driver.GetRepositoryURL())
 	},
 	PreRunE: func(cmd *cobra.Command, args []string) error {
+		if git.IsOffline() {
+			util.ExitBecauseOffline("repo")
+		}
```

Comment:
I think this should be one of the last checks (before or after validating has a driver)